src/components/Navbar.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import logo from "../Images/logo.jpg"
+import "../styles/Navbar.css"
+const Navbar = () => {
+  return (
+  <nav> 
+      <div className="wrapper">
+          <div className="left">
+                <img className="logo"src={logo}></img>
+          </div>
+          <div className="center">
+
+              <input placeholder='Search'></input>
+          </div>
+          <div className="right">
+              <ul className="right-navbar">
+                  <li className="Icons">Home </li>
```

Comment:
Suggestion: Since this are repeating, can create a component to reuse it.

Reply:
Sure will take care from next time